test/masq.test.js

```diff
@@ -7,7 +7,11 @@ window.crypto = require('@trust/webcrypto')
 
 const Masq = require('../src')
 const MasqAppMock = require('./mockMasqApp')
-const config = require('../config/config')
+const localhostConfig = require('../config/config.localhost.json')
+const testOptions = {
+  hubUrls: localhostConfig.HUB_URLS,
```

Comment:
you should edit the json file define keys you expect here: `hubUrls` and `masqAppBaseUrl`

Reply:
changed to have the same key names in Masq and config